### PR TITLE
got images to be the same size and have a clean look in uniformity

### PR DESCRIPTION
**Pull request by ChristianSmith2394**
## Description

There was an issue with UI/UX causing images and text to be out of uniform upon applying. Created uniformity to allow for a clean user experience.

#### Video Link

[Loom Video](https://www.loom.com/share/83b5d47986004519ac6447096f8b7160)

#### Trello Link

https://trello.com/c/rEbB5Yw6/332-misalignment-issues-with-mentee-and-mentor-elements

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by mikayla-kosmala** [comment]
I love it, but are we able to change the button to go underneath the text? The button its self looks scrunched as well.

**Review by BrittanyPete — CHANGES_REQUESTED** [review]
It is looking so much better! As has been mentioned though, I'm wondering if we can work on the button a little bit too. It looks small and is a little difficult to read. Not sure if it would be better under the text, or maybe the text column could be a little bit smaller, and the column with the button could be a little wider.

**Review by mikayla-kosmala — APPROVED** [review]

**Review by BrittanyPete — APPROVED** [review]

**Review by Collyngodlewski — APPROVED** [review]
After spending about an hour in discord together, I'm glad you got it figured out. Looks a lot better than it did before!

**Comment by malloryshea** [comment]
Nice job resizing and getting everything in order. As mentioned in previous comments the button is the only thing I'm having a hard time reading, maybe enlarge the button as well as the text size?

**Review by malloryshea — APPROVED** [review]

**Review by yuririnnnu — APPROVED** [review]

File: src/components/pages/RoleSignup/Signup.js [hunk 1]
```diff
@@ -31,14 +31,14 @@ const Signup = () => {
                 alt="example"
                 className="card-img"
                 src="https://images.pexels.com/photos/7504837/pexels-photo-7504837.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260"
-                style={{ height: '100%' }}
+                // style={{ height: '100%' }}
```
Comment (paulstgermain) [inline src/components/pages/RoleSignup/Signup.js:34]: We may want to remove these commented-out pieces of code down the line!

File: src/components/pages/RoleSignup/Signup.js [hunk 2]
```diff
@@ -51,19 +51,19 @@ const Signup = () => {
               <Divider
                 className="divider"
                 type="vertical"
-                style={{ height: 100 }}
+                // style={{ height: 100 }}
```
Comment (paulstgermain) [inline src/components/pages/RoleSignup/Signup.js:54]: Another exampled of commented-out code to remove later!

**Pull request merged**